src/pages/login/multi.style.js

```diff
@@ -0,0 +1,181 @@
+import { makeStyles } from "@material-ui/core";
```

Comment:
change the name of the file, multi.style is a bad name, put something like login.styles.css